spec.bs

```diff
@@ -3171,70 +3172,79 @@ might choose to require a [=k-anonymity=] threshold of fifty users over a seven
 will maintain the count over the chosen duration and compare the count to the chosen [=k-anonymity=]
 threshold when responding to [=query k-anonymity count=].
 
+The [=user agent=] must maintain a <dfn>k-anonymity cache</dfn> as a [=map=]
+whose [=map/keys=] are [=SHA-256=] hashes of the [=k-anonymity keys=] for all of the [=interest group/ads=] and [=interest group/ad components=]
+in the [=user agent=]'s [=interest group set=] and whose [=map/values=] are [=k-anonymity records=].

```

Comment:
is this wrapping too aggressive / tight?

Reply:
I'm not sure exactly what you mean here, but I adjusted it a bit.